.env.example

```diff
@@ -1,5 +1,8 @@
 ATLASSIAN_CLIENT_ID='key_ATLASSIAN_CLIENT_ID'
 ATLASSIAN_CLIENT_SECRET='key_ATLASSIAN_CLIENT_SECRET'
+AUTH_INTERNAL_ENABLED='true'
+AUTH_GOOGLE_ENABLED='true'
+AUTH_SSO_ENABLED='true'
```

Comment:
That makes sense! However when I started to implement it, I felt like from client side I was doing a lot of similar logic pattern like: `IF xxx IS enabled THEN SHOW component yyy`. It felt hard to read if I need to change it to `IF xxx IS NOT disabled THEN SHOW component yyy`.
So I change the env variables to `*_DISABLED` but I flipped them around in `*.client.config.js` and still keep `*_ENABLED`.
In server side, since we mainly use those variables to turn off some functionalities, I feel it natural to keep the `_DISABLED` naming conventions.
What do you think?